index.rst

```diff
@@ -13,13 +13,57 @@
+
+``cp_verify`` currently generates a series of plots and image views in a series of notebooks.  This is reasonably convenient for an individual (or small group) checking the quality of calibrations for a single detector, but as either the number of interested parties or detectors grow, this becomes more difficult.  We have also wanted a way to present results from the calibration construction and verification similar to the "run reports" the Camera Team produces from their [``eo_pipe``](https://github.com/lsst-camera-dh/eo_pipe) based code.  The plotting ability of ``analysis_tools`` (and the associated Plot Navigator) present an excellent way to present the things currently found in the notebooks, and by turning the numerical results into ``Metrics``, we can also track these values over time via ``Sasquatch``.
+
+
+Calibration Products Improvements
+=================================
+
+Metrics
+-------
+
+The ``cp_verify`` package should organize the quality test results (as defined in DMTN-101) that are being calculated into formal ``Metrics``.  As it is unlikely that there is an amplifier level dimension, these probably need to report the minimum, maximum, and median values found per detector, as well as the number of amplifiers that failed the test requirements.  As the variety of most of these tests is fairly limited, defining these metrics does not seem like a problem.  Generating these metrics can be added to the existing ``MergeTasks``, at the expense of ensuring that ``analysis_tools`` is a prerequisite for ``cp_verify``.  This seems like the obvious first step for this integration.
+
+Plotting
+--------
+
+Plots are currently generated in notebooks, and those that are generally useful should be migrated to use the ``analysis_tools`` plot types.  As these are written to the Butler, they can still be accessed and displayed in those notebooks.  This also seems like an easy switch, and by generating these plots as part of the appropriate pipeline tasks, they can be parallelized, reducing the complications as we switch to the full camera.
+
+Image Views
+-----------
+
+One of the main other aspects of the notebooks is the ability to flip through the set of residual images, such as a bias frame that has had only overscan and bias subtracted, to look for any remaining structure in the image that may indicate quality issues with the calibration.  As these can be displayed in matplotlib, it seems reasonable to pre-generate these as "plots" in the verification pipelines.  This is not an issue for LATISS, but constructing full focal plane images will be needed for the full camera.  The ``visualizeVisit`` pipeline can bin and mosaic the individual detectors together, but it's unclear now if that final image will be too highly binned to be useful for identifying problems.
```

Comment:
This is something that could either use the current camera image viewer, or open camera image viewer windows.